cpp/bench/sg/fil.cu

```diff
@@ -28,13 +28,14 @@
 namespace ML {
 namespace Bench {
 namespace fil {
+using namespace ML::fil::enums;
```

Comment:
the alternative to these changes is to import each enum value separately (+18 values) in `fil_test.cu`, and keep that list up to date whenever new values are added.